tests/cpp/test_spmat_coo.cc

```diff
@@ -477,3 +504,67 @@ TEST(SpmatTest, COOGetDataAndIndices) {
   _TestCOOGetDataAndIndices<int32_t>();
   _TestCOOGetDataAndIndices<int64_t>();
 }
+
+template <typename IDX>
+void _TestCOOToCSRalgs() {
```

Comment:
Keep the consistency with COOToCSRAlg.
```suggestion
void _TestCOOToCSRAlgs() {
```